dt-metrics/counters/counter-post-stats.php

```diff
@@ -25,6 +25,181 @@ public function __construct() {
         parent::__construct();
     } // End __construct()
 
+    public static function get_changed_post_counts( $post_type, $field, $start, $end, $by_month ) {
+        global $wpdb;
+
+        $added_post_changes = [];
+        $deleted_post_changes = [];
+
+        $field_type = DT_Posts::get_post_field_settings( $post_type )[$field]['type'] ?? '';
+
+        $time_unit_sql = $by_month ? 'MONTH( FROM_UNIXTIME( log.hist_time ) )' : 'YEAR( FROM_UNIXTIME( log.hist_time ) )';
+
+        switch ( $field_type ) {
+            case 'date':
+
+                $post_changes = [];
+                $post_changes_sql = [
+                    'posts.added > posts.deleted', // Added
+                    'posts.added <= posts.deleted' // Deleted
+                ];
+
+                foreach ( $post_changes_sql as $changes_sql ) {
+                    // phpcs:disable
+                    $post_changes[] = $wpdb->get_results(
```

Comment:
Nice work on this @kodinkat !
Please use $wpdb->prepare on these queries.

`log.object_note LIKE '%Added%', 1, 0` might throw an error. 

You can do something like `log.object_note LIKE '%s', 1, 0` and pass `'%Added%'` in the prepared variables.